portfolio/src/main/webapp/js/init.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Authenticates the user.
+ */
```

Comment:
nit: Authenticates or log the user out as well, right?

Reply:
Not really, it checks if the user is logged in, if yes, it returns a link for logging out, if no, a link for signing in. I think there should be "Checks if the current user is logged in" then..